src/engine/surface_load.c

```diff
@@ -733,6 +737,15 @@ void load_object_collision_model(void) {
             load_object_surfaces(&collisionData, sVertexData, TRUE);
         }
     }
+
+    f32 marioDist = o->oDistanceToMario;
+
+    // On an object's first frame, the distance is set to 19000.0f.
+    // If the distance hasn't been updated, update it now.
+    if (o->oDistanceToMario == 19000.0f) {
```

Comment:
I know the original code did this before, but it feels silly to not use `marioDist` here instead of `o->oDistanceToMario`.